src/Scripting/Core/Hosting/CommandLine/CommandLineRunner.cs

```diff
@@ -167,21 +174,29 @@ private static ScriptOptions GetScriptOptions(CommandLineArguments arguments, st
                 allowUnsafe: true,
                 checkOverflow: false,
                 warningLevel: 4,
-                parseOptions: arguments.ParseOptions);
+                parseOptions: arguments.ParseOptions,
+                createFromFileFunc: _createFromFileFunc);
         }
 
-        internal static MetadataReferenceResolver GetMetadataReferenceResolver(CommandLineArguments arguments, TouchedFileLogger loggerOpt)
+#nullable enable
```

Comment:
This looks unnecessary since we removed `#nullable disable` at the beginning of the file.